codegen/src/test/java/software/amazon/awssdk/codegen/poet/utils/SomeClientExtensionMethods.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.codegen.poet.utils;
+
+/**
+ * Test class used by {@link PoetUtilsTest}
+ */
+public class SomeClientExtensionMethods {
```

Comment:
Should we require that this be an interface?

Reply:
Hah, good point. I had mixed and matched in my tests but IRL it doesn't work if it's a class...